pw.lock

```diff
@@ -1,3 +1,3 @@
 [main]
-requirements = ["anyio==4.4.0", "blinker==1.8.2", "certifi==2024.7.4", "dep-logic==0.4.1", "distlib==0.3.8", "filelock==3.15.4", "findpython==0.6.1", "h11==0.14.0", "hishel==0.0.29", "httpcore==1.0.5", "httpx==0.27.0", "idna==3.7", "installer==0.7.0", "markdown-it-py==3.0.0", "mdurl==0.1.2", "msgpack==1.0.8", "packaging==24.1", "pbs-installer==2024.4.24", "pdm==2.16.1", "platformdirs==4.2.2", "pygments==2.18.0", "pyproject-hooks==1.1.0", "python-dotenv==1.0.1", "resolvelib==1.0.1", "rich==13.7.1", "shellingham==1.5.4", "sniffio==1.3.1", "socksio==1.0.0", "tomlkit==0.12.5", "truststore==0.9.1 ; python_version >= '3.10'", "typing-extensions==4.12.2", "unearth==0.16.0", "virtualenv==20.26.3"]
+requirements = ["anyio==4.4.0", "blinker==1.8.2", "certifi==2024.7.4", "dep-logic==0.4.1", "distlib==0.3.8", "exceptiongroup==1.2.1 ; python_version < '3.11'", "filelock==3.15.4", "findpython==0.6.1", "h11==0.14.0", "hishel==0.0.29", "httpcore==1.0.5", "httpx==0.27.0", "idna==3.7", "installer==0.7.0", "markdown-it-py==3.0.0", "mdurl==0.1.2", "msgpack==1.0.8", "packaging==24.1", "pbs-installer==2024.4.24", "pdm==2.16.1", "platformdirs==4.2.2", "pygments==2.18.0", "pyproject-hooks==1.1.0", "python-dotenv==1.0.1", "resolvelib==1.0.1", "rich==13.7.1", "shellingham==1.5.4", "sniffio==1.3.1", "socksio==1.0.0", "tomli==2.0.1 ; python_version < '3.11'", "tomlkit==0.12.5", "truststore==0.9.1 ; python_version >= '3.10'", "typing-extensions==4.12.2", "unearth==0.16.0", "virtualenv==20.26.3"]
```

Comment:
this looks sus. the lockfile seems to be different depending on what python version it was generated with, which shouldve been fixed in pyprojectx 3

Reply:
https://github.com/pyprojectx/pyprojectx/issues/110